frontend/pages/Parts/[...deviceHandleItemType].tsx

```diff
@@ -31,12 +31,14 @@ export const getServerSideProps: GetServerSideProps<AppPageProps> = async (
    );
 
    const { deviceHandleItemType } = context.params || {};
-   const [deviceHandle, ...rest] = Array.isArray(deviceHandleItemType)
+   const [deviceHandle, ...itemTypeAndRest] = Array.isArray(
+      deviceHandleItemType
+   )
```

Comment:
This seems odd. `isArray` returns a boolean, not an array, if I'm reading [the docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray) right. So I'm not sure how unpacking that works.

Reply:
@andyg0808 this condition is part of a ternary expression, so this instruction is actually returning an array 👍

Reply:
Oh, got it. Thanks.